.gitignore

```diff
@@ -74,6 +74,7 @@ web_modules/
 .env.test
 .env.production
 
+
```

Comment:
This file can be removed from the PR.

`git checkout origin/master -- .gitignore` would check out the file's content from `origin/master` branch.